buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1361,6 +1366,26 @@ def set_build_extra_options(build_specs=null) {
         if (!EXTRA_MAKE_OPTIONS) {
             EXTRA_MAKE_OPTIONS = buildspec.getVectorField("extra_make_options", SDK_VERSION).join(" ")
         }
+        if (params.CODE_COVERAGE) {
+            // avoid children pipeline set CODE_COVERAGE option multiple times
+            if (!EXTRA_MAKE_OPTIONS.contains("-DCODE_COVERAGE=ON")) {
+                // add or append CODE_COVERAGE=ON option to EXTRA_CMAKE_ARGS
+                if (EXTRA_MAKE_OPTIONS.contains("EXTRA_CMAKE_ARGS=\"")) {
+                    EXTRA_MAKE_OPTIONS = EXTRA_MAKE_OPTIONS.replace("EXTRA_CMAKE_ARGS=\"","EXTRA_CMAKE_ARGS=\"-DCODE_COVERAGE=ON ")
```

Comment:
nit: space after comma (like elsewhere)

Reply:
The space has been added. Sorry I did not notice it before. Thanks.

Reply:
Hi @keithc-ca, could you check to see if current modifications need more adjustment? Thanks a lot.